daemon/cluster/cluster.go

```diff
@@ -280,27 +280,30 @@ func (c *Cluster) GetDataPathAddress() string {
 	return ""
 }
 
-// GetRemoteAddress returns a known advertise address of a remote manager if
+// GetRemoteAddressList returns the known advertise address of a remote managers if
```

Comment:
I would say more: GetRemoteAddressList returns the known advertise address of remote managers

The Advertise address is only 1 per manager correct?